quarkus-observability-demo-activemq/quarkus-observability-demo-activemq-legume/src/test/resources/application.properties

```diff
@@ -1,9 +1,9 @@
 ## configure your datasource
-quarkus.datasource.db-kind=h2
-quarkus.datasource.jdbc.url=jdbc:h2:mem:test;MODE=PostgreSQL;DB_CLOSE_DELAY=-1
-quarkus.datasource.username: sa
-quarkus.datasource.password:
-quarkus.datasource.jdbc.max-size: 20
+#quarkus.datasource.db-kind=h2
```

Comment:
Commented out on purpose?

Reply:
there is no reactive driver support for H2